js/latoken.js

```diff
@@ -0,0 +1,635 @@
+            'cost': trade.price * trade.amount,
+            'commision': trade.commision,
+            'fee': fee,
+        };
+    }
+
+    async fetchTrades (symbol, limit = 100, params = {}) {
+        await this.loadMarkets ();
+        let market = this.market (symbol);
+        let resp = {
+            'symbol': market['symbol'],
+        };
+        let response = await this.publicGetMarketDataTrades (this.extend (resp, params));
+        let pairId = response.pairId;
+        let sym = response.symbol;
+        let tradeCount = response.tradeCount;
+        let result = [];
+        for (let i = 0; i < response.trades.length; i++) {
+            let resp = response.trades[i];
+            result.push ({
+                'pairId': pairId,
+                'symbol': sym,
+                'tradeCount': tradeCount,
+                'side': resp.side,
+                'price': resp.price,
+                'amount': resp.amount,
+                'timestamp': resp.timestamp,
+            });
+        }
+        let trades = this.parseTrades (result);
+        return trades;
+    }
+
+    async fetchMyTrades (symbol = undefined, params = {}, limit = 10) {
+        await this.loadMarkets ();
+        let request = {
+            'symbol': symbol,
+            'limit': 1000000000,
+        };
+        let response = await this.privateGetOrderTrades (this.extend (request, params));
+        let pairId = response.pairId;
+        let sym = response.symbol;
+        let tradeCount = response.tradeCount;
+        let result = [];
+        for (let i = 0; i < response.trades.length; i++) {
+            let resp = response.trades[i];
+            result.push ({
+                'pairId': pairId,
+                'symbol': sym,
+                'tradeCount': tradeCount,
+                'id': resp.id,
+                'orderId': resp.orderId,
+                'commision': resp.commision,
+                'side': resp.side,
+                'price': resp.price,
+                'amount': resp.amount,
+                'timestamp': resp.time,
+            });
+        }
+        let trades = this.parseTrades (result);
+        return trades;
+    }
+
+    parseOrderStatus (status) {
+        let statuses = {
+            'active': 'open',
+            'partially_filled': 'open',
+            'filled': 'closed',
+            'canceled': 'canceled',
+        };
+        return (status in statuses) ? statuses[status] : status;
+    }
+
+    parseOrder (order) {
+        let symbol = order['symbol'];
+        let side = order['side'];
+        let orderType = order['orderType'];
+        let price = this.safeFloat (order, 'price');
+        let amount = this.safeFloat (order, 'amount');
+        let orderStatus = order['orderStatus'];
+        let executedAmount = this.safeFloat (order, 'executedAmount');
+        let reaminingAmount = this.safeFloat (order, 'reaminingAmount');
+        let timeCreated = this.safeValue (order, 'timeCreated');
```

Comment:
This should be safeInteger (if it's an integer in milliseconds)

Reply:
done